lib/Parse/ParseDecl.cpp

```diff
@@ -1485,6 +1481,12 @@ bool Parser::parseDeclAttribute(DeclAttributes &Attributes, SourceLoc AtLoc) {
     return false;
   }
 
+  if (DK == DAK_Count && Tok.getText() == "NSKeyedArchiverClassName") {
+    DK = DAK_ObjC;
+    diagnose(Tok, diag::attr_nskeyedarchiverclassname_removed)
+      .fixItReplace(Tok.getLoc(), "objc");
```

Comment:
Yeah. I almost went as far as to not bother at all, especially since we renamed the attributes during development, but still. Fixed in the next commit, as you saw.